axum/src/extract/ws.rs

```diff
@@ -149,9 +149,37 @@ impl<F> std::fmt::Debug for WebSocketUpgrade<F> {
 }
 
 impl<F> WebSocketUpgrade<F> {
-    /// Set the size of the internal message send queue.
-    pub fn max_send_queue(mut self, max: usize) -> Self {
-        self.config.max_send_queue = Some(max);
+    /// Does nothing, instead use `max_write_buffer_size`.
+    #[deprecated]
+    pub fn max_send_queue(self, _: usize) -> Self {
+        self
```

Comment:
You say it’s deprecated upstream but that should mean we can still call it right?

Reply:
We can. However, upstream it does absolutely nothing since the send-queue concept was removed.
The field was kept & deprecated to smoothen the churn. It should be removed with the next tungstenite breaking change.

So with no reason to set it, we can just deprecate and have the method do nothing (which also even more clearly demonstrates that it does nothing to axum users). Or as a breaking change remove it.

Reply:
My 2c: Merge deprecation into `v0.6.x` (and do another point release when releasing 0.7, if not sooner), remove on `main`.

Reply:
That sounds good to me!

@alexheretic do you wanna change this PR to include the breaking changes and make another one into the `v0.6.x` branch?

Reply:
Done. I raised #2123 with the non-breaking update to `v0.6.x`.